assets/scss/_agenda.scss

```
@@ -272,6 +272,12 @@
 
 .session-card {
   background-color: #fff;
+  // Fills its grid area (grid-row: N / span rowSpan), so the card's outer
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)